gateway-messages/src/sp_to_mgs.rs

```diff
@@ -167,6 +170,38 @@ pub struct ImageVersion {
     pub version: u32,
 }
 
+#[derive(
+    Debug, Copy, Clone, PartialEq, Eq, Deserialize, Serialize, SerializedSize,
+)]
+pub enum ImageError {
+    /// Image has not been sanity checked (internal use)
+    Unchecked = 1,
```

Comment:
Do we ever care about the in-memory representation of this `enum`?  If not, I'd recommend removing the ` = 1`, because it _suggests_ that we care about the in-memory representation.

Note that the ` = 1` **does not** change how this object will be serialized.  For example:
```rust
#[derive(serde::Serialize)]
enum Foo {
    One = 1,
    Two,
}

fn main() {
    let b = bincode::serialize(&Foo::One).unwrap();
    println!("{b:?}") // prints [0, 0, 0, 0]
}
```